aiohttp/web_request.py

```diff
@@ -409,7 +409,8 @@ def post(self):
                     out.add(field.name, ff)
                 else:
                     value = yield from field.read(decode=True)
-                    if content_type.startswith('text/'):
+                    if content_type is None or \
+                            content_type.startswith('text/'):
```

Comment:
I think in case of `None`, you really cannot be sure if is it safe to decode or not. Better leave data as is.